src/features/smartSearch/components/filters/CallHistory/index.tsx

```diff
@@ -43,14 +43,18 @@ const CallHistory = ({
 }: CallHistoryProps): JSX.Element => {
   const { orgId } = useNumericRouteParams();
   const assignmentsFuture = useCallAssignments(orgId);
+  const assignments = assignmentsFuture.data || [];
+  const sortedAssignments = assignments.sort((as1, as2) => {
+    return as1.title.localeCompare(as2.title);
+  });
   const { filter, setConfig, setOp } =
     useSmartSearchFilter<CallHistoryFilterConfig>(initialFilter, {
       minTimes: 1,
       operator: CALL_OPERATOR.CALLED,
     });
 
   // only submit if assignments exist
-  const submittable = !!assignmentsFuture.data?.length;
+  const submittable = !!sortedAssignments?.length;
```

Comment:
Now that you have made the change on line 46 to save `assignmentsFuture.data || [ ]` we know that `sortedAssignments` will always be an array, so the `?` is not needed here (or below). 

The `?` operator is used if the thing to the left of the `?` can be `undefined` or `null`. In the previous code `assignmentsFuture.data?.length`, `data` can be `undefined`, so that's why the `?` was neccessary before, but is not anymore :)